zip-XXXX.rst

```diff
@@ -0,0 +1,451 @@
+them.
+
+The third requirement is the support of a more resilient ecosystem by:
+
+1. Ending the "crowding out" problem by paying development teams to work on and
+   for Zcash.
+2. Building a dev fee management structure that's resilient to the loss,
+   capture, or compromise of the Zcash Foundation.
+3. Ensuring the ecosystem can survive the loss, capture, or compromise of the
+   ECC by encouraging developer diversity and strategic input.
+
+Finally, avoid introducing unnecessary additional entities into the governance
+process.
+
+Non-requirements
+================
+
+General on-chain governance is outside the scope of this proposal. On-chain
+governance is an exciting idea -- what if we had an impartial arbiter funding
+development?  My experience with on-chain governance to date, however, leads me
+to believe it's still a risky direction. Zcash should focus on what it's good at
+-- privacy -- and leave proving on-chain governance models to other projects.
+
+While this proposal attempts to outline a long-term structure for Zcash funding
+and governance, specifying the structure beyond the next 4 years is out of
+scope.  Much will have changed in 4 years. Perhaps this structure will be
+sufficient; perhaps we'll be battling the Third Crypto War, and need to go back
+to the drawing table.
+
+Specification
+=============
+
+The below proposal is an effort to cleanly resolve the problems with Zcash's
+current governance, while
+
+* maintaining a balance of power between stakeholders
+* removing single points of failure / control
+* growing development and usage of Zcash
+* and supporting the best interests of miners, users, and developers *today*.
+
+Decentralizing development
+--------------------------
+
+A few proposals have suggested the introduction of a mysterious "third entity"
+to resolve disagreements between the Foundation and the ECC.
+
+I prefer a different approach, refocusing the role of the Foundation and making
+room for the ECC to work with a robust developer ecosystem.
+
+In this proposal, the Foundation shall support community development through
+running the forum and events, gathering community sentiment, managing short-term
+development grants, and conducting the diligence behind the assignment and
+disbursement of a development fee. This development fee shall be funded by 20%
+of the block reward, with as much as half of the fee burned in case of
+extraordinary growth in the price of ZEC.
+
+The Foundation shall receive 25% of the dev fee. If the volume-weighted average
+price of ZEC over the month means the foundation would receive greater than
+$500k that month, the Foundation shall burn enough ZEC such that their max
+benefit is
+
+.. math::
+
+  MaxBenefit(RewardDollarAmount) = Min(500000, 500000 * \sqrt{\frac{RewardDollarAmount/500000}})
+
+Capping the monthly upside of the Foundation will limit growth, while
+encouraging fiscal discipline.
```

Comment:
If the funds are spent on development (and marketing, and community management, etc) then are these funds really only to the Foundation's benefit, as implied? Aren't they really to be spent for the benefit of the whole network?

In a company, usually it's expected that operational costs increase as revenues increase to fuel growth. Why shouldn't the Zcash network similarly be able to scale up spending on the Dev Fund (and all of its funded activities) as the ZEC price increases, without this artificial ceiling capping spending on the Fund?

I see your stated reasoning:

> Capping the monthly upside of the Foundation will limit growth, while encouraging fiscal discipline.

But I don't agree that the former is desirable, or that it's worth doing in pursuit of the latter. I want to dig into the reasoning more since burning money that could otherwise be spent directly on growth activities while the network is clearly going through a growth phase (ZEC price rising substantially) seems like a very consequential decision.

Reply:
> In a company, usually it's expected that operational costs increase as revenues increase to fuel growth. Why shouldn't the Zcash network similarly be able to scale up spending on the Dev Fund (and all of its funded activities) as the ZEC price increases, without this artificial ceiling capping spending on the Fund?

We just saw what that did to the ECC, instead -- it led to unsustainable spending and worse fiscal discipline, followed by a rude awakening. This isn't revenue the ZF has "grown", regardless, so I'm not sure the analogy holds.

The idea here is that we want to keep the operation overhead of the ZF capped to promote low overhead / bureaucracy and capital efficiency. I am about to make a couple tweaks though

> I want to dig into the reasoning more since burning money that could otherwise be spent directly on growth activities while the network

About to remove the burn in favor of a longer-term purpose-restricted fund.